src/scripts/app.ts

```diff
@@ -27,8 +30,15 @@ const imgItemsArray: Array<HTMLImageElement> = [];
 
 const imageArray: Basic[] = [];
 
-const closeModal = () => {
-	if (nameInput.value) {
+// attempts
+let attemptCounter = 0;
+const maxAttempts = 2;
```

Comment:
Takie statyczne mega wartosci najlepiej pisać w formacie MAX_ATTEMPS, wtedy od razu sie to rzuca w oczy.